extensions/ohif-cornerstone-extension/src/commandsModule.js

```diff
@@ -228,6 +204,50 @@ function _getActiveViewportEnabledElement(viewports, activeIndex) {
   return activeViewport.dom;
 }
 
+function _getEnabledElement(viewports, activeIndex) {
+  const element = _getActiveViewportEnabledElement(
+    viewports.viewportSpecificData,
+    viewports.activeViewportIndex
+  );
+  if (!element) {
+    return null;
+  }
+
+  const enabledElement = cornerstone.getEnabledElement(element);
+  if (!enabledElement || !enabledElement.image) {
+    return null;
+  }
+
+  return enabledElement;
+}
+
+function _getElementToolState(element) {
+  const { toolState } = cornerstoneTools.globalImageIdSpecificToolStateManager;
+  const { imageId } = element.image;
+
+  if (toolState && toolState.hasOwnProperty(imageId)) {
```

Comment:
In general, it is preferred to use `Object.prototype.hasOwnProperty.call(toolState, imageId)` instead of `toolState.hasOwnProperty(imageId)`. Although it is unlikely, one cannot be sure that `hasOwnProperty` hasn't been overwritten somewhere in the prototype chain. This is specially true if we're dealing with an unfamiliar codebase.